stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/textinput/MessageInputView.kt

```diff
@@ -430,3 +434,7 @@ public interface MessageSendHandler {
 
     public fun editMessage(oldMessage: Message, newMessageText: String)
 }
+
+public fun interface OnMessageSendButtonClickListener {
```

Comment:
This seems to belong strictly to the class above as well, let's nest these, it adds logical scoping